RELEASE/scripts/sl_ascend.ash

```diff
@@ -833,6 +833,23 @@ item[monster] catBurglarHeistDesires()
 			wannaHeists[$monster[white lion]] = $item[lion oil];
 	}
 
+	int twinPeakProgress = get_property("twinPeakProgress").to_int();
+	boolean needStench = ((twinPeakProgress & 1) == 0);
+	boolean needFood = ((twinPeakProgress & 2) == 0);
+	boolean needJar = ((twinPeakProgress & 4) == 0);
+	boolean needInit = (twinPeakProgress == 7);
```

Comment:
Shouldn't this be `!=` instead of `==`? Or something else? You're still going to need to do the initiative part even if you aren't there yet.

Reply:
Good catch! It should be "if it's not 15"

Reply:
or even better, `needStench | needFood | needJar`